index.html

```diff
@@ -59,5 +59,170 @@ <h3>LET'S CONNECT</h3>
         </ul>
       </section>
     </main>
+    <!-- Cards section -->
+    <section class="main-container-projects">
+      <div class="cards card-1">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio.png" alt="person" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
```

Comment:
- [Optional ] please fix this weried space in the right , I suggest you to remove the padding around the about section it is not helping.
![image](https://user-images.githubusercontent.com/74744787/219780856-2757e12c-ab3b-4585-ba0e-0c0bf07ea97a.png)
![image](https://user-images.githubusercontent.com/74744787/219780948-480cdd15-5a62-497f-863d-e7862f9d3112.png)

- please fix the space in the left of the about section it should match the design
![image](https://user-images.githubusercontent.com/74744787/219782362-42cd4b47-267b-4277-b6a8-f0d503180cee.png)